core/html_api.php

```diff
@@ -642,7 +642,8 @@ function print_manage_menu( $p_page = '' ) {
 
 	# Plugins menu items - these are cooked links
 	foreach( $t_menu_options as $t_menu_item ) {
-		echo '<li>', $t_menu_item, '</li>';
+		$t_active = $p_page && strpos( $t_menu_item, $p_page ) !== false ? 'class="active"' : '';
```

Comment:
This term has been there [since grangeway added the comment back in 2009](https://github.com/mantisbt/mantisbt/commit/0d17ca9174941e550d3866f8a6c1b459c7cd6815#diff-b1c667913de013265f22c582987aa38cR879).

Looking at the code, *$t_menu_options* is populated by plugin event EVENT_MENU_MANAGE. The [documentation](http://mantisbt.org/docs/master/en-US/Developers_Guide/html-desktop/#dev.eventref.output.menu) 

> Return Value
> <Array>: List of HTML links for the management menu. 

So *cooked links* = HTML hyperlinks (`<a href="url">label</a>`).